proto/viam/app/v1/app.proto

```diff
@@ -1411,3 +1423,77 @@ message CreateKeyFromExistingKeyAuthorizationsResponse {
   string id = 1;
   string key = 2;
 }
+
+message OrganizationSetLogoRequest {
+  string org_id = 1;
+  bytes logo = 2;
+}
+message OrganizationSetLogoResponse {}
+
+message OrganizationGetLogoRequest {
+  string org_id = 1;
+}
+message OrganizationGetLogoResponse {
+  string url = 1;
+}
+
+message EnableAuthServiceRequest {
+  string org_id = 1;
+}
+message EnableAuthServiceResponse {}
+
+message DisableAuthServiceRequest {
+  string org_id = 1;
+}
+message DisableAuthServiceResponse {}
+
+message CreateOAuthAppRequest {
+  string org_id = 1;
+  string name = 2;
+  OAuthConfig oauth_config = 3;
+}
+message CreateOAuthAppResponse {
+  string id = 1;
+  string secret = 2;
+}
+
+message ReadOAuthAppRequest {
+  string org_id = 1;
+  string id = 2;
+}
+message ReadOAuthAppResponse {
+  string name = 1;
+  string secret = 2;
+  OAuthConfig oauth_config = 3;
+}
+
+message UpdateOAuthAppRequest {
+  string org_id = 1;
+  string id = 2;
+  string name = 3;
```

Comment:
application_id and application_name?

Reply:
Sure -- whats the deal with this stuff considering it was approved in scope that way?

Reply:
(I'm sure fine but asking to be safe)

Reply:
ive been told that its fine to tweak api that was posed in scope as long as its minimal

Reply:
In practice its a client ID, which will make the most sense to a customer who is trying to debug their auth flows because they will see the client ID in their browser's nav bar. Similarly, prepending "client_" to "secret" so its clear we're returning a client secret on creation.

Lmk what you think.